### ISSUE TITLE
Additional minor customizations to dominos cast bar

### ISSUE THREAD

**Issue by addonaddict** [tullamods/Dominos#494]
**Is your feature request related to a problem? Please describe.**
I love the addon but I tend to always disable the cast bar module of dominos (so do some of my friends who use it) and I would love if we could style the cast bar a little bit further. It is very close to being perfect after recent updates to it, but there are still some things I would love to be included in the options. All the other cast bar addons are unnecessarily bulky with trillions of options that are nothing but clutter on your screen, dominos does it best for me.

**Describe the solution you'd like**
List of things I would like to be able to adjust from most important to least important:
-option toggle to show cast bar ticks for channeled spells like "drain life" for warlock
-specify color of cast bar (using RGB #'s maybe?) - this could be done like 1 color for all, or 1 color for each type of spell (contextual coloring option in dominos but with ability to set a color for each type)
-specify background of cast bar (bit too transparent for me currently)
-specify type of border around cast bar, similar to how other addons do it (ex: set border to match blizzard tooltip)
-specify font size (this honestly is probably for all dominos bars, not just cast bar. I would love an option to adjust font size such as for key #, macro name, spell stacks, etc., as currently I only see the option to change font)
-modify "show spark" to show similar to how other cast bar addons do it, instead of only gradiant coloring

-last one is not really a user adjustment but i feel as the cast bar fades out too fast when spell is interrupted, if possible i would like the spell bar to stay active for about 2x as long when the spell was interrupted to give time for the user to notice it (especially helpful since I multibox and like to see the castbar turn red on my other screens which instantly tells me something is wrong like a herb was not picked up)

Thank you so much, and dont take it like I want you to add all of the above to the addon. This is purely just my feedback on things I would love to see that would benefit me

**Comment by Tuller** [comment]
Yeah, some of these will be implemented. I'm not sure about the spark one, though.

**Comment by arcadepro** [comment]
Yeah, I too wouldn't mind having the default border art back rather than the tooltip border.
